docs/source/performance.md

```diff
@@ -0,0 +1,75 @@
+# Performance tuning
+
+1.6 Scylla Operator adds a new experimental feature allowing users to optimize Kubernetes nodes
+
+## Node tuning
+
+From 1.6 a new CRD called NodeConfig is available which allows users to target Nodes which should be tuned.  
+Nodes matching the provided placement conditions will be subject for tuning.
+
+Below example NodeConfig tunes nodes having `scylla.scylladb.com/node-type=scylla` label:
+```
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: NodeConfig
+metadata:
+ name: cluster
+spec:
+ placement:
+   nodeSelector:
+     scylla.scylladb.com/node-type: scylla
+```
+For more details about new CRD use:
+```
+kubectl explain nodeconfigs.scylla.scylladb.com/v1alpha1
+```
+
+Tuning consists of two separate optimizations: common tuning of node, and tuning based on Scylla Pods and their resource assignment.
+First optimization is executed when tuning Pod lands on Node. Second one is executed when Scylla Pod lands on the same Node.  
+Tuning Pods are created in special namespace called `scylla-operator-node-tuning`
+
+Only Scylla Pods having Guaranteed Quality of Service class are tuned, so make sure your ScyllaCluster meets all necessary conditions.
+
+## Kubernetes tuning
+
+By default, the kubelet uses the CFS quota to enforce pod CPU limits.  
+When the node runs many CPU-bound pods, the workload can move around different CPU cores depending on whether the pod 
+is throttled and which CPU cores are available.
+However, kubelet may be configured to assign CPUs exclusively, by setting the CPU manager policy to static.
+
+Consult your Kubernetes provider how to set it up, because it's configurable on kubelet startup.
+
+Pods which receive a Guaranteed Quality of Service class can take advantage of this option. 
+As Guaranteed pods whose containers fit the requirements for being statically assigned are scheduled to the node, 
+CPUs are removed from the shared pool and placed in the cpuset for the container.
+
+In our case there are two requirements each ScyllaCluster must fulfill in order to receive a Guaranteed QoS class:
+* resource request and limits must be equal or only limits have to be provided
+* agentResources must be provided and their requests and limits must be equal, or only limits have to be provided
+
+An example of such a ScyllaCluster which receives a Guaranteed QoS class is below:
+
+```
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: guaranteed-cluster
+  namespace: scylla
+spec:
+  version: 4.5.1
+  agentVersion: 2.5.2
+  datacenter:
+    name: us-east-1
+    racks:
+    - name: us-east-1a
+      members: 3
+      storage:
+        capacity: 500Gi
+      agentResources:
+        limits:
+          cpu: 1
+          memory: 1G
+      resources:
+        limits:
```

Comment:
maybe it would be good to set request explicitly so it's seen they are the same

Reply:
done